packages/dev/docs/pages/react-aria/testing.mdx

```diff
@@ -0,0 +1,109 @@
+or wait for the required time (real timers). This is required for certain interactions (e.g. long press) that some of the patterns support.
+
+Once the `User` is initialized, you can use its `createTester` method to initialize a specific ARIA pattern tester in your test cases. This gives you access to that pattern's specific utilities that you can then call
+within your test to query for specific subcomponents or simulate common interactions. `createTester` requires two arguments, the first being the name of the ARIA pattern tester you are creating and the second being a set of initialization options specific to that
+pattern, typically including the `root` element (e.g. the menu trigger button, table, etc). See [below](#patterns) for more details on what is supported for each individual ARIA pattern tester.
+
+```ts
+// YourTest.test.ts
+import {User} from '@react-aria/test-utils';
+
+// Provide whatever method of advancing timers you use in your test, this example assumes Jest with fake timers
+let testUtilUser = new User({interactionType: 'mouse', advanceTimer: jest.advanceTimersByTime});
+// ...
+
+it('my test case', async function () {
+  // Render your test component/app and initialize the table tester
+  render();
+  let table = testUtilUser.createTester('Table', {root: screen.getByTestId('test_table')});
+  // ...
+});
+```
+
+See below for the full definition of the `User` object.
+
+<ClassAPI links={testUtilDocs.links} class={testUtilDocs.exports.User} />
+
+### Patterns
+
+Below is a list of the ARIA patterns testers currently supported by `createTester`. See the accompanying component testing docs pages for a sample of how to use
+the testers in your test suite.
+
+-  [React Aria Components ComboBox](ComboBox.html#test-utils) and [React Spectrum ComboBox](../react-spectrum/ComboBox.html#test-utils)
+
+-  [React Aria Components GridList](GridList.html#test-utils) and [React Spectrum ListView](../react-spectrum/ListView.html#test-utils)
+
+-  [React Aria Components ListBox](ListBox.html#test-utils) and [React Spectrum ListBox](../react-spectrum/ListBox.html#test-utils)
+
+-  [React Aria Components Menu](Menu.html#test-utils) and [React Spectrum MenuTrigger](../react-spectrum/MenuTrigger.html#test-utils)
+
+-  [React Aria Components Select](Select.html#test-utils) and [React Spectrum Picker](../react-spectrum/Picker.html#test-utils)
+
+-  [React Aria Components Table](Table.html#test-utils) and [React Spectrum TableView](../react-spectrum/TableView.html#test-utils)
+
+-  [React Aria Components Tabs](Tabs.html#test-utils) and [React Spectrum Tabs](../react-spectrum/Tabs.html#test-utils)
+
+-  [React Aria Components Tree](Tree.html#test-utils) and [React Spectrum TreeView](../react-spectrum/TreeView.html#test-utils)
```

Comment:
Do we want to copy the content of the test util section over to the RSP testing page as well? Then we could just link to the aria pages here and the rsp pages there?